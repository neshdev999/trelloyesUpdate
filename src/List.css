.listContainer {
    display: flex;
    align-items: center;
    justify-content: flex-start;
    flex-direction: column;
    background-color: #dfe3e6;
    border-radius: 5px;
    height: 100vh;
    overflow-y: scroll;
}

.listItem {
    flex: 1;
}