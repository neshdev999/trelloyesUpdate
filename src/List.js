import React from 'react';
import './List.css';
import Card from './Card';

class List extends React.Component{

    cardIds = this.props.listCardIds;
    listHeader = this.props.listHeader;
    
    render(){
        return(
            <div className="listContainer">
                <div className="listItem">
                    <h2>{this.listHeader}</h2>
                    <div>
                        {this.cardIds.map((item, index) => (
                         <div key={index}><Card id={item} cardStore={this.props.listStore} /></div>
                        ))}
                    </div>
                </div>
                <div>
                    <button>+ Add Random Card</button>
                </div>
            </div>
        )
    }
}


export default List;