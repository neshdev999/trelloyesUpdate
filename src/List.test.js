import React from 'react';
import ReactDOM from 'react-dom';
import List from './List';
import Store from './store';
import renderer from 'react-test-renderer';


/* Smoke Tests */
describe('First List renders correctly', ()=>{
    it('List renders without crashing', () =>{
        const div = document.createElement('div');
        const currentStore = Store;
        ReactDOM.render(<List listStore={currentStore} listId={currentStore.lists[0].id} listHeader={currentStore.lists[0].header} listCardIds={currentStore.lists[0].cardIds} key={1} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('Second List renders correctly', ()=>{
    it('List renders without crashing', () =>{
        const div = document.createElement('div');
        const currentStore = Store;
        ReactDOM.render(<List listStore={currentStore} listId={currentStore.lists[1].id} listHeader={currentStore.lists[1].header} listCardIds={currentStore.lists[1].cardIds} key={2} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('Third List renders correctly', ()=>{
    it('List renders without crashing', () =>{
        const div = document.createElement('div');
        const currentStore = Store;
        ReactDOM.render(<List listStore={currentStore} listId={currentStore.lists[2].id} listHeader={currentStore.lists[2].header} listCardIds={currentStore.lists[2].cardIds} key={3} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('Fourth List renders correctly', ()=>{
    it('List renders without crashing', () =>{
        const div = document.createElement('div');
        const currentStore = Store;
        ReactDOM.render(<List listStore={currentStore} listId={currentStore.lists[3].id} listHeader={currentStore.lists[3].header} listCardIds={currentStore.lists[3].cardIds} key={4} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});

/* Snapshot Tests */

describe('First List UI renders as expected', ()=>{
    it('renders the List UI as expected', () =>{
        const currentStore = Store;    
        const tree = renderer.create(<List listStore={currentStore} listId={currentStore.lists[0].id} listHeader={currentStore.lists[0].header} listCardIds={currentStore.lists[0].cardIds} key={1} />).toJSON();
        expect(tree).toMatchSnapshot();     
    });
});

describe('Second List UI renders as expected', ()=>{
    it('renders the List UI as expected', () =>{
        const currentStore = Store;    
        const tree = renderer.create(<List listStore={currentStore} listId={currentStore.lists[1].id} listHeader={currentStore.lists[1].header} listCardIds={currentStore.lists[1].cardIds} key={2} />).toJSON();
        expect(tree).toMatchSnapshot();     
    });
});

describe('Third List UI renders as expected', ()=>{
    it('renders the List UI as expected', () =>{
        const currentStore = Store;    
        const tree = renderer.create(<List listStore={currentStore} listId={currentStore.lists[2].id} listHeader={currentStore.lists[2].header} listCardIds={currentStore.lists[2].cardIds} key={3} />).toJSON();
        expect(tree).toMatchSnapshot();     
    });
});

describe('Fourth List UI renders as expected', ()=>{
    it('renders the List UI as expected', () =>{
        const currentStore = Store;    
        const tree = renderer.create(<List listStore={currentStore} listId={currentStore.lists[3].id} listHeader={currentStore.lists[3].header} listCardIds={currentStore.lists[3].cardIds} key={4} />).toJSON();
        expect(tree).toMatchSnapshot();     
    });
});

