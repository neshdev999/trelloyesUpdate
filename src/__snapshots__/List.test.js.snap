// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First List UI renders as expected renders the List UI as expected 1`] = `
<div
  className="listContainer"
>
  <div
    className="listItem"
  >
    <h2>
      First list
    </h2>
    <div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                First card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Second card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Fifth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Sixth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Seventh card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Tenth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Twelfth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Thirteenth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <button>
      + Add Random Card
    </button>
  </div>
</div>
`;

exports[`Fourth List UI renders as expected renders the List UI as expected 1`] = `
<div
  className="listContainer"
>
  <div
    className="listItem"
  >
    <h2>
      Fourth list
    </h2>
    <div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Twelfth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Thirteenth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <button>
      + Add Random Card
    </button>
  </div>
</div>
`;

exports[`Second List UI renders as expected renders the List UI as expected 1`] = `
<div
  className="listContainer"
>
  <div
    className="listItem"
  >
    <h2>
      Second list
    </h2>
    <div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Second card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Third card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Fourth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Sixth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Eighth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Ninth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Eleventh card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <button>
      + Add Random Card
    </button>
  </div>
</div>
`;

exports[`Third List UI renders as expected renders the List UI as expected 1`] = `
<div
  className="listContainer"
>
  <div
    className="listItem"
  >
    <h2>
      Third list
    </h2>
    <div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                First card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Second card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Third card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Fourth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Fifth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Sixth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Seventh card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Eighth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Ninth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Tenth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Eleventh card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Twelfth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="cardContainer"
        >
          <div
            className="cardContentItem cardContentFirstItems"
          >
            <div
              className="cardContentFirstItemsPeta"
            >
              <h3
                className="cardHeader"
              >
                Thirteenth card
              </h3>
            </div>
            <div
              className="cardContentFirstItemsPeta"
            >
              <button
                className="cardButton"
              >
                delete
              </button>
            </div>
          </div>
          <div
            className="cardContentItem"
          >
            <div
              className="cardContentSecondItem"
            >
              <p
                className="cardContent"
              >
                lorem ipsum
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <button>
      + Add Random Card
    </button>
  </div>
</div>
`;
